### ISSUE TITLE
Bug report: Premature reveal of a monster ability card when opening a room

### ISSUE THREAD

**Issue by nrnrnr** [Tarmslitaren/FrosthavenAssistant#179]
(Apologies in advance for the slightly cryptic nature of this report.  I've been forgetting to file it for weeks, and if I wait until the big box is unpacked, I'm afraid I'll forget again.)

In playing a scenario I will have to look up later, we opened a door and were instructed to read section 40.2 in the section book.  This door triggered new special rules which required counting from the round in which the door was opened.  The door also revealed enemies.

When I tapped on the `#40.2` box, Assistant correctly asked for the standee numbers of the revealed enemies.  But it also started a new round, including turning a new monster ability card for one of the enemies already on the map.  The behavior I expected was to leave the exposed ability card(s) untouched, and to expose a new card only for monster types that (a) were revealed in the new room *and* (b) were not already on the map.   And I expected the new "round 1" to start only at the *end* of the round in which the door was opened.

If you want more details, please let me know and I'll dig out my notes and see what I can add.

----
TL;DR:  Regardless of special rules, opening a door and revealing enemies should not start a new round.

**Comment by nrnrnr** [comment]
Followup: The scenario in question was Frosthaven #46 (Dead Pass).  We opened the door with Shrike Fiends still alive in the first room.  Tapping `#40.2` not only added the revealed enemies (expected) but also gave the Shrike Fiends a new ability card (unexpected) and started a new round (unexpected).

**Comment by Tarmslitaren** [comment]
Fixed in 1.8.4. And as always: thanks for reporting!
